docs/Contribution.md

```diff
@@ -0,0 +1,104 @@
+# Contribution Guide
+
+Thank you for your interest in contributing to Mobile App Control Center, Systers! There are always bugs to file; bugs to fix in the code; improvements to be made to the documentation; and more.
+
+
+The instructions given below are for software developers who want to work on [our current code](https://github.com/systers/macc).
+
+## Git workflow
```

Comment:
How about this order?
1. What to work on
2. Coding standards(styles)
3. Git workflow